python/pyspark/sql/tests/test_pandas_udf.py

```diff
@@ -197,6 +197,66 @@ def foofoo(x, y):
             ).collect
         )
 
+    def test_pandas_udf_detect_unsafe_type_conversion(self):
+        from distutils.version import LooseVersion
+        from pyspark.sql.functions import pandas_udf
```

Comment:
nit: after the unit test reorg, the `pandas_udf` import is at the top so it's not needed here or in the other test

Reply:
Removed it. Thanks!